main.go

```diff
@@ -29,9 +30,7 @@ func rssHandler(rw http.ResponseWriter, r *http.Request) {
 		Created:     time.Now(),
 	}
 
-	for _, feed := range sourceFeeds {
-		fetch(feed, master)
-	}
+	fetchFeeds(master)
```

Comment:
Since the number of feeds is defined waitgroup may not be needed. Could just loop and grab from channel?

```go
itemChan := make(chan *feeds.Item)

for _, feed := range sourceFeeds {
  go fetch(feed, itemChan)
}

for i := 1; i < len(sourceFeeds); i++ {  
  master.Add(<-itemChan)
}
```

And then thread the `itemChan` through the fetch instead of master

```go
func fetch(feed sourceFeed, itemChan chan *feeds.Item) {
```

```go
itemChan <- &feeds.Item{
  Title:       stripPodcastEpisodePrefix(items[i].Title),
  Link:        &feeds.Link{Href: items[i].Links[0].Href},
  Description: items[i].Description,
  Author:      &feeds.Author{Name: sourceName},
  Created:     published,
}
```